packages/block-editor/src/components/simulate-media-query/index.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * External dependencies
+ */
+import { filter, get } from 'lodash';
+import { match } from 'css-mediaquery';
+
+/**
+ * WordPress dependencies
+ */
+import { useEffect } from '@wordpress/element';
+
+const ENABLED_MEDIA_QUERY = '(min-width:0px)';
+const DISABLED_MEDIA_QUERY = '(min-width:999999px)';
+
+const VALID_MEDIA_QUERY_REGEX = /\((min|max)-width:[^\(]*?\)/g;
+
+function getStyleSheetsThatMatchHostname() {
```

Comment:
Any reasons to return early from this function if `window` is `undefined`?

Reply:
Good point. This code shouldn't ever run on the server side, but better safe than sorry I guess 😅